teos/watcher.py

```diff
@@ -70,6 +71,7 @@ def __init__(self, db_manager, block_processor, responder, sk_der, max_appointme
         self.max_appointments = max_appointments
         self.expiry_delta = expiry_delta
         self.signing_key = Cryptographer.load_private_key_der(sk_der)
+        self.last_known_block = db_manager.load_last_block_hash_responder()
```

Comment:
Should this be db_manager.load_last_block_hash_watcher()?

Reply:
Indeed, bad copy paste :(